examples/jax/debugging/simple_spread/feedforward/decentralised/run_mappo.py

```diff
@@ -88,19 +89,20 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
 
     # Create the system.
     system = mappo.MAPPOSystem()
-
+    system.update(MonitorExecutorEnvironmentLoop)
```

Comment:
I like this as this clears shows a case when we do not want PPO system to have recording but the specific run file that add the feature :smile:

Reply:
Does this mean that our example will now always record episodes? If so, I think it would be better to make it a separate example.